EUR-Lex.js

```diff
@@ -1,21 +1,21 @@
 {
 	"translatorID": "bf053edc-a8c3-458c-93db-6d04ead2e636",
 	"label": "EUR-Lex",
-	"creator": "Philipp Zumstein",
-	"target": "^https?://(www\\.)?eur-lex\\.europa\\.eu/(legal-content/[A-Z][A-Z]/TXT/|search.html\\?)",
+	"creator": "Philipp Zumstein, Pieter van der Wees",
+	"target": "^https?://(www\\.)?eur-lex\\.europa\\.eu/(legal-content/[A-Z][A-Z]/(TXT|ALL)/|search.html\\?)",
 	"minVersion": "3.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2017-12-21 20:27:07"
+	"lastUpdated": "2021-01-13 13:04:33"
 }
 
 /*
 	***** BEGIN LICENSE BLOCK *****
 
-	Copyright © 2017 Philipp Zumstein
+	Copyright © 2017-2020 Philipp Zumstein, Pieter van der Wees
```

Comment:
To the extent that copyright notices make any difference whatsoever (approximately: not at all), it's actually better to keep the range. It would be inaccurate and misleading to claim just the current year. (The conservative advice is to include each year separately, but that's too annoying. A range is a reasonable compromise.)

Reply:
That's fair. More correct, we just don't do it that way in many translators.

Reply:
Could we start cataloging informal norms like this somewhere, both in terms of coding style (that isn't already covered by eslint) and metadata?

Reply:
I have updated the year and kept the range as per @dstillman's request.